scripts/upgrade

```diff
@@ -178,6 +178,11 @@ ynh_script_progression --message="Upgrading dependencies..."
 
 ynh_install_app_dependencies $pkg_dependencies
 
+if ynh_version_gt "1.0.3~ynh1" "${previous_version}" ; then
+	ynh_script_progression --message="Mobilizon v1.0.3: Remove previous NodeJS"
+	ynh_remove_nodejs
```

Comment:
If another app uses Nodejs, wouldn't that impact it?

Reply:
It will only remove current nodejs if no more used:

https://github.com/YunoHost/yunohost/blob/1279755463100e6c4a0ba7d4b50f8f39658c8eac/data/helpers.d/nodejs#L187-L207

Reply:
Perfect, thanks for the follow up 👍